frameworks/kafka/tests/test_availability.py

```diff
@@ -13,7 +14,7 @@
 
 from tests import config
 
-BROKERS_KILL_GRACE_PERIOD = int(os.environ.get('BROKER_KILL_GRACE_PERIOD', 30))
+BROKER_KILL_GRACE_PERIOD = int(os.environ.get('BROKER_KILL_GRACE_PERIOD', 30))
```

Comment:
@paegun below in `setup_module()` we refer to `BROKERS_KILL_GRACE_PERIOD`, which is probably a typo.